api/src/main/java/uk/gov/hmcts/reform/divorce/caseapi/model/CcdCallbackResponse.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.divorce.caseapi.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.Map;
+
+@Data
+@NoArgsConstructor
```

Comment:
do you need this if you have @Builder? I can't remember

Reply:
will remove you are right we don't need this annotation